posthog/settings.py

```diff
@@ -731,3 +732,10 @@ def show_toolbar(request):
 
 # keep in sync with plugin-server
 EVENTS_DEAD_LETTER_QUEUE_STATSD_METRIC = "events_added_to_dead_letter_queue"
+
+VERSION_REQUIREMENTS = [VersionRequirement(service="postgres", version_range=">=14.0.0,<15.0.0",)]
```

Comment:
@guidoiaquinti @tiina303 what do we want to have here? (reminder that these are soft requirements for now)

Reply:
I think version range is 👍 

Nit: `version_range` -> `supported_version` (as technically we could also support a single version and not a range)

Reply:
Yeah I was in the dilemma of "version" not covering when we use ranges, and "range" not covering when we use a specific version.

Will update though.